src/components/BurgerMenu.jsx

```diff
@@ -13,15 +13,15 @@ export const BurgerMenu = () => {
         </Link>
       </h2>
 
-      {/* <Link className="menu-item" to="/?topic=cooking">
+      <Link className="menu-item" to="/?topic=cooking">
```

Comment:
it might make more sense from a user's point of view if the url  for these was, for example, `/articles/cooking` rather than having the API query in there.